releasenotes/notes/remove-deprecated-gate-args-b29ba17badcb77ca.yaml

```diff
@@ -0,0 +1,7 @@
+---
+deprecations:
+  - |
+    The deprecated keyword arguments for the circuit gates ``q``,
+    ``ctl*`` and ``tgt*`` have been removed. Instead, only the names
```

Comment:
Can you include the release this was deprecated in?

Reply:
Done in a432a5c 👍